docs/data/data-grid/server-side-data/ServerSideDataGrid.js

```diff
@@ -28,24 +29,24 @@ export default function ServerSideDataGrid() {
     [fetchRows],
   );
 
-  const initialStateWithPagination = React.useMemo(
+  const initialState = React.useMemo(
```

Comment:
"Pagination" was removed from the name but is still present in the object's value, should it be removed?

Reply:
I removed "Pagination" from the name to shorten it. With the perception that we could not add `WithFeatureName` for every entry added to the `initialState`. The structure should be evident in explaining what it contains.

Reply:
Ok. `initialState` and `initState` are semantically equivalent, not a great naming scheme, but it probably doesn't matter much for a doc example.

Reply:
Nice observation, totally agreed. I think an inline version would be better in that case since we don't need to memoize it either. Updated: https://github.com/mui/mui-x/pull/16359/commits/e11e639088779f1e1c913d48d501dbde8ec3688c